modules/apps/jaggery/emm/src/emm/units/appbar/public/js/appbar.js

```diff
@@ -77,7 +77,7 @@ function loadNotifications(){
     var serviceURL = "/mdm-admin/notifications/NEW";
 
     var successCallback = function (data) {
-        if(data){
+        if(!data){
             data = "[]";
```

Comment:
Why we are passing a quoted array here in the form of a string instead of simply passing data = [];